slave.mk

```diff
@@ -891,6 +891,9 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 	PASSWORD="$(PASSWORD)" \
 	TARGET_MACHINE=$($*_MACHINE) \
 	IMAGE_TYPE=$($*_IMAGE_TYPE) \
+	SONIC_ENABLE_IMAGE_SIGNATURE="$(SONIC_ENABLE_IMAGE_SIGNATURE)" \
+	SONIC_CETIFICATE_PATH="$(SONIC_CETIFICATE_PATH)" \
+	TARGET_PATH="$(TARGET_PATH)" \
```

Comment:
Design macros same way as parameters.
Also check [https://github.com/Azure/sonic-buildimage/blob/master/rules/config](https://github.com/Azure/sonic-buildimage/blob/master/rules/config) for conventions.

Reply:
Add the description in the config file.